web/src/beta/components/TabMenu/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { FC, useState, ReactNode } from "react";
+
+import Icon from "@reearth/beta/components/Icon";
+import Icons from "@reearth/beta/components/Icon/icons";
+import { styled } from "@reearth/services/theme";
+
+interface TabObject {
+  icon: keyof typeof Icons;
+  component: ReactNode;
+}
+
+type Props = {
+  title: string;
+  tabs: Record<string, TabObject>;
+};
+
+const TabMenu: FC<Props> = ({ tabs, title }) => {
+  const [selected, setSelected] = useState<string | null>(null);
```

Comment:
Can you move this out of this component and instead have 2 props: 
```
selectedTab: string;
onSelectedTabChange: (tab: string) => void;
```
I think we will want to know which tab is selected outside of this component for knowing what to query/etc

Reply:
Done. 
Just for the `stories part` to dynamically update the tab when the user clicks on the icon, I have used `useArgs` , reference from [this blog](https://medium.com/urbandataanalytics/changing-args-values-state-in-storybook-without-usestate-hook-15cc096716f7), I hope this is the correct implementation as per our standards. I couldn't find a similar example that existed in the current codebase.

Reply:
Should be fine for now, though I think you can write your own args (not quite sure how this useArgs is working).
<img width="288" alt="Screenshot 2023-08-17 at 16 01 58" src="https://github.com/reearth/reearth/assets/34051327/eb7ead31-b196-4998-a59b-c38ebc2b484d">

Reply:
it's so that `onSelectedTabChange` function can work.